source/portable/os/ota_os_freertos.c

```diff
@@ -145,10 +144,13 @@ OtaOsStatus_t OtaReceiveEvent_FreeRTOS( OtaEventContext_t * pEventCtx,
     {
         otaOsStatus = OtaOsEventQueueReceiveFailed;
 
-        LogError( ( "Failed to receive event from OTA Event Queue: "
-                    "xQueueReceive returned error: "
-                    "OtaOsStatus_t=%i ",
-                    otaOsStatus ) );
+        if( retVal != errQUEUE_EMPTY )
```

Comment:
xQueueReceive either returns `pdTRUE` if there is an item is retrieved from queue or `pdFALSE` if there is no item for a timeout period. So I think checking for `errQUEUE_EMPTY` is not useful for this API.

Reply:
Removed the condition and add a config for polling timeout to avoid printing lots of error log. Set the polling timeout to 1s by default.